components/script/dom/webgl_extensions/ext/webglvertexarrayobjectoes.rs

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use core::cell::Ref;
+use core::iter::FromIterator;
+use dom::bindings::cell::DOMRefCell;
+use dom::bindings::codegen::Bindings::WebGLVertexArrayObjectOESBinding;
+use dom::bindings::js::{JS, MutNullableJS};
+use dom::bindings::js::Root;
+use dom::bindings::reflector::{Reflector, reflect_dom_object};
+use dom::globalscope::GlobalScope;
+use dom::webglbuffer::WebGLBuffer;
+use dom_struct::dom_struct;
+use std::cell::Cell;
+use std::collections::HashMap;
+use webrender_traits::WebGLVertexArrayId;
+
+#[dom_struct]
+pub struct WebGLVertexArrayObjectOES {
+    reflector_: Reflector,
```

Comment:
ouch, yes It should be added to this one because it's a different object (not the extension itself). Shouldn't missing the WebGLObject give a compilation error?